### PR TITLE
enh(Resources/Details/TimeLine/Events):  rework timeline UI

### PR DESCRIPTION
**Pull request by Noha-ElAbrouki**
enhancement timeLine tab Detail 
**some screens :** 

> dark mode

TimeLineDetailsDark

> light mode

timeLineDetails

### CODE REVIEW

**Comment by Thebarda** [comment]
You got some eslint issues

**Comment by Noha-ElAbrouki** [comment]
@Wedierre

**Comment by Noha-ElAbrouki** [comment]
@Wedierre

File: www/front_src/src/Resources/Details/tabs/Timeline/Event.tsx [hunk 1]
```diff
@@ -69,12 +69,26 @@ const getTypeIds = (): Array<string> => {
...
     padding: theme.spacing(1),
   },
+  eventTimeLineContainer: {
+    alignItems: 'center',
+    display: 'flex',
+    flexGrow: 0.5,
+    flexWrap: 'wrap',
+    justifyContent: 'space-between',
+    marginBottom: theme.spacing(1),
+    padding: theme.spacing(0, 1, 0, 1),
```
Comment (Thebarda) [inline www/front_src/src/Resources/Details/tabs/Timeline/Event.tsx:89]:
```suggestion
    padding: theme.spacing(0, 1),
```

File: www/front_src/src/Resources/Details/tabs/Timeline/Events/index.tsx [hunk 2]
```diff
@@ -117,22 +132,26 @@ const Events = ({ timeline, infiniteScrollTriggerRef }: Props): JSX.Element => {
...
+                    <span className={classes.divisionSubtitle}>
+                      <Typography display="inline">
+                        {areStartAndEndDateEqual
+                          ? format({
+                              date: (
+                                startDate?.(formattedLocale) as Dayjs
+                              )?.toISOString(),
+                              formatString: 'LL',
+                            })
+                          : formattedDivisionDates.join(' ')}
```
Comment (Thebarda) [inline www/front_src/src/Resources/Details/tabs/Timeline/Events/index.tsx:151]: You can extract it to a specific variable

**Review by Wedierre — DISMISSED** [review]

**Review by Jordan91 — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Comment by Wedierre** [comment]
OK pour moi

**Pull request merged**